eclair-core/src/test/scala/fr/acinq/eclair/payment/relay/ChannelRelayerSpec.scala

```diff
@@ -74,7 +74,7 @@ class ChannelRelayerSpec extends ScalaTestWithActorTestKit(ConfigFactory.load("a
 
   /** we parse logs to make sure the listener has received the event and prevent race conditions */
   def publishUpdate(e: LocalChannelUpdate) = {
-    LoggingTestKit.debug(s"updating local channel info for channelId=${e.channelId} shortChannelId=${e.shortChannelId}").expect {
+    LoggingTestKit.debug(s"updating local channel info for channelId=${e.commitments.channelId} shortChannelId=${e.shortChannelId}").expect {
```

Comment:
Please revert (and same elsewhere, why did you add these unrelated and unnecessary changes?)

Reply:
My dad, just did not pay attention to what exactly was there before, will revert.